Modules/Scripted/WebServer/Resources/docroot/browse.html

```diff
@@ -47,7 +54,55 @@
           },
         ],
       },
+      extensions: [
+        {
+          id: "slicerExtension",
+          getToolbarModule() {
+            return {
+              definitions: [
+                {
+                  id: "slicer",
+                  label: "3D Slicer",
+                  icon: "exclamation-triangle",
```

Comment:
This looks like an error. I would recommend to use the `cube`, `edit`, or `bars` icon instead.

```suggestion
                  icon: "cube",
```